pkg/genswagger/main.go

```diff
@@ -76,16 +78,210 @@ func main() {
 	if err != nil {
 		log.Fatal(err.Error())
 	}
-	if err := ioutil.WriteFile(*outputFile, jsonBytes, 0644); err != nil {
+	if err := os.WriteFile(*outputFile, jsonBytes, 0644); err != nil {
 		log.Fatal(err.Error())
 	}
 }
 
-func createConfig() *common.Config {
-	return &common.Config{
-		CommonResponses: map[int]spec.Response{
+func allOperations(path *spec3.Path) []*spec3.Operation {
+	// all operations (see the definition of path.PathProps):
+	return []*spec3.Operation{
+		path.Get,
+		path.Put,
+		path.Post,
+		path.Delete,
+		path.Options,
+		path.Head,
+		path.Patch,
+		path.Trace,
+	}
+}
+
+// find all the schemas in the spec and clear uniqueItems for non-array types
+func clearNonArrayUniqueItems(swagger *spec3.OpenAPI) {
+	for _, path := range swagger.Paths.Paths {
+		for _, param := range path.Parameters {
+			if param == nil {
+				continue
+			}
+			if schema := param.Schema; schema != nil {
+				if !schema.Type.Contains("array") {
+					schema.UniqueItems = false
+				}
+			}
+		}
+		for _, op := range allOperations(path) {
+			if op == nil {
+				continue
+			}
+			// find all the schemas in the spec
+			for _, param := range op.Parameters {
+				if param != nil {
+					if schema := param.Schema; schema != nil {
+						if !schema.Type.Contains("array") {
+							schema.UniqueItems = false
+						}
+					}
+				}
+			}
```

Comment:
just keep it consistent

```suggestion
			for _, param := range op.Parameters {
				if param == nil {
					continue
				}
				if schema := param.Schema; schema != nil {
					if !schema.Type.Contains("array") {
						schema.UniqueItems = false
					}
				}
			}
```